dxr/templates/folder.html

```diff
@@ -27,14 +27,20 @@
     <thead>
       <tr>
         <th scope="col">Name</th>
+        {% for header in plugin_headers %}
+          <th scope="col">{{ header }}</th>
+        {% endfor %}
         <th scope="col">Modified</th>
```

Comment:
Just so you know, "Modified" is kind of useless atm, since it reflects age on disk and not age in the VCS.